english/doug-adams/arthur-dent.md

```diff
@@ -0,0 +1,11 @@
+You find yourself, still dressed in pajamas and robe, lying in the dirt in
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.